sdk/eventhubs/azure_messaging_eventhubs/README.md

```diff
@@ -157,80 +157,127 @@ More information about Event Hubs features and terminology can be found here: [l
 Examples for various scenarios can be found on in the samples directory in our GitHub repo for
 [Event Hubs](https://github.com/Azure/azure-sdk-for-rust/tree/main/sdk/eventhubs/azure-messaging-eventhubs/samples).
 
-## Send events
-
-The following example shows how to send events to an event hub:
+## Open an eventhubs message producer on an eventhubs instance.
```

Comment:
Isn't the product name "Event Hubs"? We should make sure our public docs follow branding guidelines e.g., ".NET" not ".Net", "Key Vault", not "KeyVault", etc.